uk.co.compucorp.civicrm.hrleaveandabsences/js/src/leave-absences/leave-type-wizard/components/leave-type-wizard-fields.html

```diff
@@ -1,6 +1,6 @@
-<div class="row form-horizontal">
+<div class="row form-horizontal" ng-repeat="tab in section.tabs" ng-show="tab.active">
```

Comment:
Is it possible to we use `ng-if` here? which might improve the performance even more.

Reply:
done.